internal/controller/postgrescluster/apply.go

```diff
@@ -64,3 +76,30 @@ func (r *Reconciler) apply(ctx context.Context, object client.Object) error {
 	}
 	return err
 }
+
+// applyPodSecurityContext is called by apply to work around issues with server-side apply.
```

Comment:
:question: Do we have the specific SSA issue captured anywhere?

Reply:
Not that I know of. I looked for a specific issue at k8s that might explain the difference between v1.18 and v1.19, ~but I couldn't find one~. Found it: https://pr.k8s.io/92661

In v1.18.15, sending a `null` in a slice field causes the field/key to be dropped from `managedFields` without changing the value on the object.

I also saw this on an older version of v1.18 when we added then [removed](https://github.com/CrunchyData/postgres-operator/commit/b7aba1e4d99e739087a4feee4fb399f1190b3e6c) the wrapper script from command/args on some containers. As I recall, we deferred at the time because we were adding more than removing and not yet concerned with breaking changes.

Reply:
Added a link to the upstream issue. [`45138beb`](https://github.com/CrunchyData/postgres-operator/pull/2593/commits/45138beb5dbe1257257de55c109e5e693fdf752a)